airstack-modules/modules/airstack/domain-name/domain-name.ts

```diff
@@ -751,32 +691,34 @@ export namespace domain {
    * @dev this is a generic function to create ids for entities
    * @param transactionHash transaction hash
    * @param blockHeight block number in the chain
-   * @param logIndex txn log index
+   * @param logOrCallIndex txn log or call index
    * @returns entity id in string format
    */
-  function createEntityId(transactionHash: string, blockHeight: BigInt, logIndex: BigInt): string {
-    return transactionHash.concat("-").concat(blockHeight.toString()).concat('-').concat(logIndex.toString());
+  function createEntityId(transactionHash: string, blockHeight: BigInt, logOrCallIndex: BigInt): string {
+    return transactionHash.concat("-").concat(blockHeight.toString()).concat('-').concat(logOrCallIndex.toString());
   }
 
   /**
+   * @dev this function needs to be removed
```

Comment:
Removed?

Reply:
removed as this function is no longer being used

Reply:
removed makeSubnode function also